docs/integrations/paloalto.md

```diff
@@ -2,55 +2,130 @@ uuid: 903ec1b8-f206-4ba5-8563-db21da09cafd
 name: Palo Alto
 
 ## Overview
+
 Palo Alto Networks offers an enterprise cybersecurity platform which provides network security, cloud security, endpoint protection, and various cloud-delivered security services.
 
-## Setup
-This setup guide will show you how to forward logs produced by your Palo Alto firewalls to SEKOIA.IO by means of an rsyslog transport channel.
+## I. Palo Alto logs
```

Comment:
Title numbers are now automatically added in CSS, no longer needed to explicitly add them.

BTW, The full name of the company is “Palo Alto Networks”. I don’t know the full name of their product.